Domains/DocumentMetadata/DocumentMetadata.ecschema.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="DocumentMetadata" alias ="dmd" version="01.00.00" description="Metadata from a Document Management System" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
+    <ECSchemaReference name="CoreCustomAttributes" version="01.00.03" alias="CoreCA"/>
+    <ECSchemaReference name="BisCore" version="01.00.14" alias="bis"/>
+
+    <ECCustomAttributes>
+        <ProductionStatus xmlns="CoreCustomAttributes.01.00.00">
+            <SupportedUse>NotForProduction</SupportedUse>
+        </ProductionStatus>
+    </ECCustomAttributes>
+
+    <ECEntityClass typeName="DocumentLink" modifier="None" displayLabel="Document link" description="Link to a named persistent set of information, retrievable from within a file, system or application storage hierarchy.">
+        <BaseClass >bis:RepositoryLink</BaseClass>

```

Comment:
I am working on the connector code and I am starting to question the merits of deriving the class versus a relationship that adds metadata to a repository element. The repository link element is used widely and is used in relationships etc. Changing the type while preserving elementId is riskier.